email_campaigns/thanks-mentor-email.md

```diff
@@ -0,0 +1,47 @@
+{{ name }},
+
+# Thanks for mentoring at the Mentored Sprint for Diverse Beginners at PyCon US 2019!
+
+<insert photo?> (not sure if possile)
+
+We want to take the time to thank each and everyone of you for joining us, for being
+excellent, and for contributing to this amazing community. You've inspired many new
+open source contributors, and the Python community is better because of you
+💜
+
+This is our first time organizing the mentored sprint and numerous people
+have asked when will we do it again! Before we can get to that we'd like to hear
+about your experience as a project mentor.
+
+We are including an anonymous survey to hear your thoughts on what we can do even
```

Comment:
I would make the language here even stronger. Proofreads welcome, but something like:

"Your feedback is invaluable to improving future iterations of this program. Please take a few moments to share your thoughts and let us know what improvements we can make to provide an even better experience for participants."